packages/badge/stories/badge.stories.ts

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import '../sp-badge.js';
+import { html, TemplateResult } from '@spectrum-web-components/base';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-checkmark-circle.js';
+
+export default {
+    title: 'Badge',
+    component: 'sp-badge',
+};
+
+export const Default = (): TemplateResult => {
+    return html`
+        <sp-badge>Badge</sp-badge>
+    `;
+};
+
+export const Icons = (): TemplateResult => {
+    return html`
+        <sp-badge>No icon</sp-badge>
+
+        <sp-badge>
+            <sp-icon-checkmark-circle slot="icon"></sp-icon-checkmark-circle>
+            Icon and label
+        </sp-badge>
+
+        <sp-badge>
+            <sp-icon-checkmark-circle slot="icon"></sp-icon-checkmark-circle>
+        </sp-badge>
+    `;
+};
+
+export const Sizes = (): TemplateResult => {
```

Comment:
Having multiple sizes in a single story can make VRTs difficult with changes, but this is pretty contained, so it might not be an issue.